sklearn/naive_bayes.py

```diff
@@ -191,6 +195,9 @@ def _update_mean_variance(n_past, mu, var, X):
         var: array-like, shape (number of Gaussians,)
             Variances for Gaussians in original set.
 
+        sample_weight : array-like, shape (n_samples), optional
+            Weights applied to individual samples (1. for unweighted).
```

Comment:
Shouldn't it be " ``None`` for uniform weights"? Not sure what the docstring usually says ^^